### PR TITLE
speedup

### PR DESCRIPTION
**Pull request by yhung119**
- update deps to fix build, update test API key, use same persister between runs
- Fixup Dockerfile

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -38,9 +37,9 @@ func LoadConfigs(configName string) *PeacemakrConfig {
 		log.Printf("Error reading config, %v", err)
 		log.Println("Using default values instead")
 		configuration = PeacemakrConfig{
-				Verbose: true,
+				Verbose: false,
```
Comment (notasecret) [inline main.go:40]: many thx

File: main.go [hunk 2]
```diff
@@ -38,9 +37,9 @@ func LoadConfigs(configName string) *PeacemakrConfig {
...
 				Host: "https://api.peacemakr.io",
-				PersisterFileLocation: "/tmp/",
+				PersisterFileLocation: "/tmp/.peacemakr",
```
Comment (notasecret) [inline main.go:42]: I am wondering, maybe this should be in the home directory of the current user? tmp sometimes gets blown away :)

**Review by notasecret — APPROVED** [review]
woooo

File: main.go (hunk 2 above)
Comment (yhung119) [reply main.go:42]: this is configurable in the yml file :)

**Pull request merged**